README.md

```diff
@@ -77,24 +77,24 @@ docker compose up -d --no-build
 🛒 Distribution
 ------------------
 
-Ready to use binaries are available [from GitHub](https://github.com/green-code-initiative/ecoCode-csharp-sonarqube/releases).
+Ready to use binaries are available [from GitHub](https://github.com/green-code-initiative/creedengo-csharp-sonarqube/releases).
 
 🧩 Compatibility
 -----------------
 
-| Plugin version | SonarQube version   | EcoCode nuget package version |
-| -------------- | ------------------- | ----------------------------- |
-| 1.1.0+         | 9.9.+ LTS to 10.5.1 | 1.1.0                         |
+| Plugin version | SonarQube version   | creedengo nuget package version |
+| -------------- | ------------------- |---------------------------------|
+| 1.1.0+         | 9.9.+ LTS to 10.5.1 | 1.1.0                           |
```

Comment:
please add a new line with 2.0.0 version (it will be the next release with renaming work included)